pallets/dkg-metadata/src/lib.rs

```diff
@@ -608,6 +612,10 @@ impl<T: Config> OneSessionHandler<T::AccountId> for Pallet<T> {
 	where
 		I: Iterator<Item = (&'a T::AccountId, T::DKGId)>,
 	{
+		let changed = changed;
+		#[cfg(test)]
```

Comment:
This is really bad design. We shouldn't add such config flags into a runtime module like this.